include/LightGBM/dataset.h

```diff
@@ -739,6 +754,36 @@ class Dataset {
     return raw_data_[numeric_feature_map_[feat_ind]].data();
   }
 
+  inline void store_mapping()  {
+    // store maps to mapping
+    int offset = 0;
+    for(int i=0; i < maps.size(); i++){    
+      std::string name;
+      
+      if(i == label_idx_){
+        name = "label";
+        offset--;
+      }else{ 
+        name = feature_names_[i + offset];
+        }
+        
+      name = name + '\n';
```

Comment:
>name = name + '\n'; [](http://example.com/codeflow?start=6&length=19)

I see you are using += when you process string, which is good. But why not this one? :-)